### PR TITLE
Add Stride latest

### PR DESCRIPTION
**Pull request by ghost**
This pr add a the cask for the new atlassian chat application Stride.

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/stride.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+cask 'stride' do
+  version '1.6.20'
+  sha256 '0af358efe5f7016fbefb5f28f8b4c1d9438b5b134b4fc6bd7ce188e7924a4c16'
+
+  # stride-desktop-downloads.s3.amazonaws.com/releases/mac was verified as official when first introduced to the cask
+  url 'https://stride-desktop-downloads.s3.amazonaws.com/releases/mac/dmg/64/prod/Stride.dmg'
```
Comment (commitay) [inline Casks/stride.rb]: How did you get the URL?

Is there a public webpage with the download link?

**Review by commitay — DISMISSED** [review]

File: Casks/stride.rb (hunk 1 above)
Comment (bestit-tkellner) [reply Casks/stride.rb]: Woaw. You're fast :-D

You can find this downloads here: https://www.stride.com/downloads

**Comment by bestit-tkellner** [comment]
@commitay This is my first cask, thanks for your help!

I've cleanend up my branch history / commit.

**Comment by commitay** [comment]
No problem, thanks for the quick response on the `url`!

> I've cleanend up my branch history / commit.

GitHub allows us to squash when merging, we prefer that contributors don't squash commits so we can see changes that have been made.

**Comment by bestit-tkellner** [comment]
> GitHub allows us to squash when merging, we prefer that contributors don't squash commits so we can see changes that have been made.

Ah okay. Habit from other projects.

**Review by commitay — APPROVED** [review]

**Pull request merged**